<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>180</y>
      <width>74</width>
      <height>36</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>72</width>
       <height>34</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>节点</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>节点类型</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>图片文件</string>
        </property>
        <item>
         <property name="text">
          <string>分组节点</string>
         </property>
         <item>
          <property name="text">
           <string>图片节点</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>分组节点2</string>
         </property>
         <item>
          <property name="text">
           <string>图片节点2</string>
          </property>
         </item>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFolder"/>
   <addaction name="actionAddFiles"/>
   <addaction name="actionDeleteItem"/>
   <addaction name="actionScanItems"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomRealSize"/>
   <addaction name="actionZoomFitWidth"/>
   <addaction name="actionZoomFitHeight"/>
   <addaction name="separator"/>
   <addaction name="actionDockVisible"/>
   <addaction name="actionDockFloat"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionAddFolder">
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="toolTip">
    <string>添加文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小图片</string>
   </property>
  </action>
  <action name="actionZoomRealSize">
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="toolTip">
    <string>图片实际大小显示</string>
   </property>
  </action>
  <action name="actionZoomFitWidth">
   <property name="text">
    <string>适合宽度</string>
   </property>
   <property name="toolTip">
    <string>适合宽度显示图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionDeleteItem">
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="toolTip">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本系统</string>
   </property>
  </action>
  <action name="actionZoomFitHeight">
   <property name="text">
    <string>适合高度</string>
   </property>
   <property name="toolTip">
    <string>适合高度显示图片</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionScanItems">
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="toolTip">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actionDockVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>窗口可见</string>
   </property>
   <property name="toolTip">
    <string>停靠窗口可见</string>
   </property>
  </action>
  <action name="actionDockFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>窗口浮动</string>
   </property>
   <property name="toolTip">
    <string>停靠窗口浮动</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
